microsoft/surface/kernel/default.nix

```diff
@@ -1,5 +1,16 @@
 { config, lib, pkgs, ... }:
 
-{
-  boot.kernelPackages = pkgs.callPackage ./linux-5.19.17 { };
+let
+  inherit (lib) mkOption types;
+
+in {
+  imports = [
+    ./linux-5.19.17
+    ./linux-6.0.11
```

Comment:
Do you think you can maintain two kernel versions of this?

Reply:
> Do you think you can maintain two kernel versions of this?
> Otherwise it would be also nice to have just one kernel that is properly maintained.

TBH, I'm expecting this to be a temporary situation, as it seems like the 6.0 series is very nearly stable.

In this case, it seems like there's also been a period of instability on the 5.19 and 6.0 series, and while the 6.0.x series is definitely going to be the more capable one, it also seems to have changed pretty fast, so made me a little hesitant.

But I should've added an explanation of why I refactored the underlying code in this way to the prev. PR, so I'll do that, too.

If you prefer, I can delay a little and update this to just "switch" ?

Reply:
should be good.